web_ui/src/pages/annotator/components/sidebar/dataset/dataset-list-item-details.component.tsx

```diff
@@ -44,7 +44,7 @@ export const DatasetListItemDetails = ({
     const id = getMediaId(mediaItem);
 
     const columns: ComponentProps<typeof Grid>['columns'] = useMemo(() => {
-        const initialColumns = ['var(--width)', minmax('size-1000', '1fr')];
+        const initialColumns = ['size-800', minmax('size-1000', '1fr')];
```

Comment:
Can you clarify this change?

Reply:
Yes, the new grid doesn't rely on CSS variables anymore. Instead, it uses the configuration passed to the `MediaList` component